WordPress/Classes/Models/Person.swift

```diff
@@ -90,7 +92,7 @@ struct Person {
             }
         }
         
-        static let roles : [Role] = [.SuperAdmin, .Admin, .Editor, .Author, .Contributor]
+        static let roles = [Role.Admin, .Editor, .Author, .Contributor]
```

Comment:
Could we leave a comment here stating why `.SuperAdmin` is omitted?  At first glance I thought it was a mistake.